.vscode/launch.json

```diff
@@ -80,9 +130,9 @@
             "args": [
                 "force:lightning:local:device:start",
                 "-p",
-                "iOS",
+                "ios",
                 "-t",
-                "<replace with udid for sfdxdebug device>"
+                "3627EBD5-E9EC-4EC4-8E89-C6A0232C920D"
```

Comment:
was this guid for your local debugging?

Reply:
Yes but it serves as an example so anyone who wants to run locally would understand that they just need to update this UDID with another one (doesn't need to be from a simulator named `sfdxdebug` but rather any simulator)